rapid7_insightidr/help.md

```diff
@@ -36,6 +36,50 @@ Example input:
 
 ### Actions
 
+#### Close Investigations in Bulk
+
+This action is used to close all investigations that match the parameters.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|alert_type|string|None|False|The category of alerts that should be closed|None|Account Created|
+|datetime_from|string|None|False|An ISO formatted timestamp, default last week|None|2018-07-01 00:00:00|
+|datetime_to|string|None|False|The category of alerts that should be closed, when empty use now|None|2018-07-01 00:00:00|
+|max_investigations_to_close|integer|None|False|The category of alerts that should be closed|None|10|
```

Comment:
The variable name doesn't match the description. 

Is this a category or a limit?